Examples/VimeoUpload-iOS/VimeoUpload-iOS/ViewControllers/MyVideosViewController.swift

```diff
@@ -156,26 +156,7 @@ class MyVideosViewController: UIViewController, UITableViewDataSource, UITableVi
     
     func cellDidRetryUploadDescriptor(cell: VideoCell, descriptor: UploadDescriptor)
     {
-        let videoUri = descriptor.uploadTicket.video!.uri!
-
-        self.retry(uploadDescriptor: descriptor, completion: { [weak self] (error) in
-         
-            guard let strongSelf = self else
-            {
-                return
-            }
-            
-            if error != nil
-            {
-                return
-            }
-            
-            // Reload the cell so that it reflects the state of the newly retried upload
-            if let indexPath = strongSelf.indexPath(for: videoUri)
-            {
-                strongSelf.tableView.reloadRows(at: [indexPath], with: .none)
-            }
-        })
+        // TODO: Implement retry
```

Comment:
Would it be worth using `RetryUploadOperation` here?

Reply:
@nicolelehrer I would, but I'll leave that to another ticket. Previously there was some retry logic using it but it was all commented out which was probably for a reason, and I didn't want to re-implement all of that.

Reply:
@nicolelehrer I would, but I'll leave that to another ticket. Previously there was some retry logic using it but it was all commented out which was probably for a reason, and I didn't want to re-implement all of that.